models/src/main/java/si/fri/tabletop/places/Place.java

```diff
@@ -22,10 +22,16 @@
 
     private String description;
 
-    private Date submitted;
+    @Column()
```

Comment:
Is `@Column` annotation needed here? You do not have it with `String address`.

Reply:
Wasn't sure if we want placeType (column = "place_type") as in "bar", "restaurant", etc.
Forgot to remove it, gg wp.